compiler/resolution/functionResolution.cpp

```diff
@@ -2462,6 +2466,50 @@ static bool resolveBuiltinCastCall(CallExpr* call)
   return false;
 }
 
+static bool resolveClassBorrowMethod(CallExpr* call) {
+  if (isUnresolvedSymExpr(call->baseExpr)) {
+    if (call->numActuals() == 2) { //mt, this
+      if (call->isNamed("borrow") && dtMethodToken == call->get(1)->typeInfo()) {
```

Comment:
Let's make this do pointer comparison on the UnresolvedSymExpr* -> unresolved. To do that, you'll add a `const char* astrBorrow;` in symbol.h / symbol.c and initialize it just like e.g. astrPostfixBang

Reply:
I got the creating and comparing agains `astrBorrow` part. And I think you want me to convert the `call->baseExpr` to `UnresolvedSymExpr` and call `isNamed` on it?

Reply:
You can call isNamedAstr or `myUnresolvedSymbol->unresolved == myAstr`

Reply:
Ah, didn't know `unresolved` was a field. Will do

Reply:
I liked isNamedAstr better.